pt-BR/form_helpers.md

```diff
@@ -343,181 +343,180 @@ When parsing POSTed data, Rails will take into account the special `_method` par
+<option value="2">Madri</option>
+<option value="3">Berlim</option>
 ```
 
-The first argument to `options_for_select` is a nested array where each element has two elements: option text (city name) and option value (city id). The option value is what will be submitted to your controller. Often this will be the id of a corresponding database object but this does not have to be the case.
+O primeiro argumento para `options_for_select` é um *array* aninhado onde cada elemento tem dois elementos: texto da opção (nome da cidade) e valor da opção (id da cidade). O valor da opção é o que será enviado ao seu *controller*. Freqüentemente, esse será o id de um objeto de banco de dados correspondente, mas não precisa ser o caso.
 
-Knowing this, you can combine `select_tag` and `options_for_select` to achieve the desired, complete markup:
+Sabendo disso, é possível combinar `select_tag` e `options_for_select` para obter a marcação completa desejada:
 
 ```erb
 <%= select_tag(:city_id, options_for_select(...)) %>
 ```
 
-`options_for_select` allows you to pre-select an option by passing its value.
+`options_for_select` permite pré-selecionar uma opção passando seu valor.
 
 ```html+erb
-<%= options_for_select([['Lisbon', 1], ['Madrid', 2], ['Berlin', 3]], 2) %>
+<%= options_for_select([['Lisboa', 1], ['Madri', 2], ['Berlim', 3]], 2) %>
 ```
 
-Output:
+Saída:
 
 ```html
-<option value="1">Lisbon</option>
-<option value="2" selected="selected">Madrid</option>
-<option value="3">Berlin</option>
+<option value="1">Lisboa</option>
+<option value="2" selected="selected">Madri</option>
+<option value="3">Berlim</option>
 ```
 
-Whenever Rails sees that the internal value of an option being generated matches this value, it will add the `selected` attribute to that option.
+Sempre que o Rails vê que o valor interno de uma opção sendo gerada corresponde a este valor, ele adicionará o atributo `selected` aquela opção.
 
-You can add arbitrary attributes to the options using hashes:
+É possível adicionar atributos arbitrários às opções usando hashes:
 
 ```html+erb
 <%= options_for_select(
   [
-    ['Lisbon', 1, { 'data-size' => '2.8 million' }],
-    ['Madrid', 2, { 'data-size' => '3.2 million' }],
-    ['Berlin', 3, { 'data-size' => '3.4 million' }]
+    ['Lisboa', 1, { 'data-size' => '2.8 million' }],
+    ['Madri', 2, { 'data-size' => '3.2 million' }],
+    ['Berlim', 3, { 'data-size' => '3.4 million' }]
   ], 2
 ) %>
 ```
 
-Output:
+Saída:
 
 ```html
-<option value="1" data-size="2.8 million">Lisbon</option>
-<option value="2" selected="selected" data-size="3.2 million">Madrid</option>
-<option value="3" data-size="3.4 million">Berlin</option>
+<option value="1" data-size="2.8 million">Lisboa</option>
+<option value="2" selected="selected" data-size="3.2 million">Madri</option>
+<option value="3" data-size="3.4 million">Berlim</option>
 ```
 
-### Select Boxes for Dealing with Model Objects
+### *Select Boxes* Caixas de seleção para lidar com objetos de modelo
```

Comment:
```suggestion
### Caixas de Seleção (*Select Boxes*) com Objetos *Model*
```

Não sei se ficou muito bom mas achei um pouco confuso a parte do "lidar" até o inglês soa um pouco estranho o que acha?

Não costumamos traduzir o termo **model** só colocar em itálico por ser um termo técnico. [No manual de tradução](https://github.com/campuscode/rails-guides-pt-BR/blob/master/TRANSLATION_MANUAL.md#gloss%C3%A1rio) tem alguns outros termos também, se quiser dar uma olhada.